docs/en_US/Compressor/Pruner.md

```diff
@@ -92,3 +92,48 @@ You can view example for more information
 
 ***
 
+## FPGM Pruner
+FPGM Pruner is an implementation of paper [Filter Pruning via Geometric Median for Deep Convolutional Neural Networks Acceleration](https://arxiv.org/pdf/1811.00250.pdf)
+
+Previous works utilized “smaller-norm-less-important” criterion to prune filters with smaller norm values in a convolutional neural network. In this paper, we analyze this norm-based criterion and point out that its effectiveness depends on two requirements that are not always met: (1) the norm deviation of the filters should be large; (2) the minimum norm of the filters should be small. To solve this problem, we propose a novel filter pruning method, namely Filter Pruning via Geometric Median (FPGM), to compress the model regardless of those two requirements. Unlike previous methods, FPGM compresses CNN models by pruning filters with redundancy, rather than those with “relatively less” importance.
+
+### Usage
+First, you should import pruner and add mask to model.
+
+Tensorflow code
+```python
+from nni.compression.tensorflow import FPGMPruner
+config_list = [{
+    'pruning_rate': 0.5,
+    'op_types': ['Conv2D']
```

Comment:
what types does this pruning algorithm support?

Reply:
Support Conv1D, and Conv2D for now.

Reply:
better to make it clear in this doc

Reply:
Thanks, document updated, but there is no list of supported op_types in document, because the list maybe updated later.  If specified op_type is not supported, there will be error messages.